src/models/diluent.js

```diff
@@ -32,8 +32,10 @@ module.exports = (sequelize, DataTypes) => {
     }
   );
 
-  /* Diluent.associate = function(models) {
-    // associations can be defined here
-  };*/
+  Diluent.associate = function(models) {
+    this.hasMany(models.RecipesDiluents, {
+      foreignKey: 'diluentId'
+    });
+  };
```

Comment:
There's also a hasMany relationship to PreparationsDiluents, right?

Reply:
I'm not seeing a table preparations_diluents, was that added recently?

Reply:
Good catch, added in #11 !